spec/models/edm/web_resource_spec.rb

```diff
@@ -67,11 +67,26 @@
 
     subject { edm_web_resource }
 
-    context 'when the file is of type image' do
+    context 'when the file is of type image(jpeg)' do
```

Comment:
Let's DRY this up and ensure that we always cover all allowed content types with:
```ruby
EDM::WebResource::ALLOWED_CONTENT_TYPES.each do |content_type|
  context "when the file is of type #{content_type}" do
    let(:mime_type) { content_type }
    it { is_expected.to be_valid }
  end
end
```